### PR TITLE
[#20][docs] Update docs structure for pulsar-hub sync

### LINKED ISSUE #20: [Doc]--cloud storage sink connector doc
as the connector doc uses a new structure, more inputs are required. Please help provides the inputs as i comment in this google doc:
https://docs.google.com/document/d/1xxAtu-GjTLnzY0s6lQ3f_UJnhUDk6pULGPpThMFS47g/edit#

You can take SQS source connector doc for your reference.
SQS source connector doc: https://github.com/streamnative/pulsar-io-sqs/blob/master/docs/sqs-source.md

### PR DESCRIPTION
**Pull request by alpreu**
Fixes #20 

### Motivation

The document structure for hub.streamnative.io has changed and needed to be updated.

### Modifications

Updated the doc to the new structure

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

### Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (not applicable / **docs**/ JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: docs/io-cloud-storage-sink.md [hunk 1]
```diff
@@ -1,58 +1,52 @@
...
-id: "io-cloud-storage-sink"
+dockerfile: "https://hub.docker.com/r/streamnative/pulsar-io-cloud-storage"
+alias: Cloud Storage Sink Connector
 ---
 
-Cloud Storage Sink connector for Pulsar
+The Cloud Storage sink connector supports exporting data from Pulsar topics to cloud storage (such as AWS S3 and Google GCS) either in Avro, JSON, Parquet or other formats. According to your environment, the Cloud Storage sink connector can guarantee exactly-once support for exporting data to cloud storage.
 
-The Cloud Storage Sink connector supports exporting data from Pulsar Topic to cloud storage in Avro, json, parquet and other formats, such as aws-s3, google-GCS, etc. According to your environment, the Cloud Storage sink connector can guarantee exactly-once support for exporting to cloud storage.
+![](/images/connectors/cloud-storage-sink.png)
```
Comment (alpreu) [inline docs/io-cloud-storage-sink.md:8]: This image does not exist yet

**Review by danpi — APPROVED** [review]

File: docs/io-cloud-storage-sink.md (hunk 1 above)
Comment (Huanli-Meng) [reply docs/io-cloud-storage-sink.md:8]: I can help to draw the image, and the image link should be changed to "/docs/cloud-storage-sink.png".

**Review by Huanli-Meng — COMMENTED** [review]
leave my two comments. PTAL

File: docs/io-cloud-storage-sink.md (hunk 1 above)
Comment (alpreu) [reply docs/io-cloud-storage-sink.md:8]: Thanks for helping out with the image @Huanli-Meng. I will change the link and wait to add the image then

**Review by Huanli-Meng — APPROVED** [review]
LGTM

**Pull request merged**